### DISCUSSION TITLE
Any way we can wtach on one video source while using the remote configured on other video source?

### DISCUSSION THREAD

**Discussion by f520k510** [GerryDazoo/Slinger#629] [General]
My slinger works well on my rasphberry pi.  The slingbox is a 500.  For some reason, if the stb is turned off, then it won't powered on.  I think it is due to some software issue or slingbox contraint.  The stb is connected via hdmi to the 500.  On my 350, slinger works well and  we can power the stb on and off with no problem.  I , therefore, want to make a slingbox profile on slinger config.ini with the 500.  I want to watch on composite, or component, but use the remote on the hdmi, so I can power on the stb.  Is there any way to di it?

Here is the log when I try to connect to 500 with the stb powered off.

 * Debug mode: off
Broadcast Responder Running
Server IP address is 192.168.1.200
Received GETSERVERCONFIG Request b'GETSERVERCONFIG' ('192.168.1.8', 49457)
ServerConfig port=65432,maxstreams=6,urlbase=chuliu|
GETSLINGBOXIDS Request
10/05/2023, 22:04:29.509   Streaming request from ('192.168.1.8', 10900)  /chuliu/4k
STREAM=192.168.1.8:10900:0

Slinginfo  350/500 12 30 192.168.1.25 5201 3072 1 vNor7FnMLa82QZR
4k Starting Stream for  192.168.1.8:10900
4k Opening Control stream 0x2000 0 0
Connecting... ('192.168.1.25', 5201) Control
4k Selecting Video Source 2
4k Keycodes= [1, 4, 5, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 24, 25, 26, 27, 28, 29, 30, 31, 33, 35, 36, 37, 38, 39, 40, 41, 42, 45, 46, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77]
VideoParameters: Resolution= 12 FrameRate= 30 VideoBandwidth= 2000 VideoSmoothness= 0 IframeRate= 5 AudioBitRate= 64
Connecting... ('192.168.1.25', 5201) Stream
Traceback (most recent call last):
  File "/home/slingbox/slingbox_server.py", line 836, in streamer
    s_ctl, stream  = start_slingbox_session(streams)
  File "/home/slingbox/slingbox_server.py", line 428, in start_slingbox_session
    first_buffer = bytearray(stream.recv(pksize, socket.MSG_PEEK))
socket.timeout: timed out
4k Badness starting slingbox session  timed out None
Streamer:  4k Waiting for first stream, flushing any IR requests that arrive while not connected to slingbox

**Comment by GerryDazoo** [comment]
Sorry for the late response. Yes you can do this. In your config.ini set  RemoteCode=1 to tell the server to use the remote control that is configured for video source 1 (HDMI).

**Comment by f520k510** [comment]
Do you mean put RemoteCode=1 line in the [server] section, or under each slingbx?
Also, I think videosource 0,1,2 are composite, component, and hdmi respectively.
Why did you say that RemoteCode=1 will use hdmi?

**Reply by GerryDazoo** [reply]
Yes put it in each of the slingbox sections. You are correct the RemoteCode needs to be 2 for HDMI. I mis-cut and pasted.

**Reply by f520k510** [reply]
Thanks for reply.  I am sorry for the late reply.  I was afraid if I had posted the result, it might lead to more work to be done for Gerry.
After all, it is a special need on my side, and normal users don't use it this way.
But after second thought, I would post it here and explained the situation.  If Gerry has some spare time, he can help me.  If not, then it's ok. 

Here is the result:  I created a dummy slingbox section.  I made the videosource=1 and remotecode=2.  I can connect to this instance of slinger server with the stb powered off.  Note I cannot connect to the instance with videosource=2, or hdmi with the stb powered off.

Now, both instances connects fine with stb powered on.  The only explanation I can come up with is that due to some reason, slinger won't connect to hdmi, but will connect to component.  The remotecode parameter doesn't seems to work, unfortunately.  The remote on the dummy instance fails to power on the stb, unfortunately.

I asked my friend on the slinger site to install a device called rm4 mini, a ir blaster capable of sending ir commands whcih are triggered remotely with a smartphone app.  I then use the app to power on the stb, then I can connect to slinger with hdmi as videosource.

It works but I hope to not having to install another device when we have the ir balster with a 500 slingbox.

Therefere, Gerry, please investigate if you have spare time.  Thanks.  Or maybe I did something wrong in my little experiment.  Do let me know.  Thanks.

**Comment by f520k510** [comment]
Hi, gerry and everyone else.
Just to report that the issue of not being able to login and turn on slingbox 500 has been resolved by reverting back to slinger 4.0.
I just read the info on a forum and google search for 4.0.

I think I will stick to 4.0 for now.
Thanks.
